framework/src/bundle/BundlePrivate.cpp

```diff
@@ -251,7 +205,8 @@ Bundle::State BundlePrivate::GetUpdatedState(LockType& l)
   return static_cast<Bundle::State>(state.load());
 }
 
-void BundlePrivate::SetStateInstalled(bool sendEvent, UniqueLock& resolveLock)
+void BundlePrivate::SetStateInstalled(bool sendEvent,
```

Comment:
I suggest removing the lock parameter completely from all the functions that used to need it because of BundleThreads.